pom.xml

```diff
@@ -71,8 +71,8 @@ under the License.
   <properties>
     <javaVersion>8</javaVersion>
     <mdoVersion>2.2.0</mdoVersion>
-    <mavenVersion>3.2.5</mavenVersion>
-    <slf4jVersion>1.7.5</slf4jVersion>
+    <mavenVersion>3.9.5</mavenVersion>
```

Comment:
Do we need such update for this exercises?

Reply:
No ,is irrelevant